### DISCUSSION TITLE
Losing hardware DCC due to USB-C to HDMI connection?

### DISCUSSION THREAD

**Discussion by mirkanu** [MonitorControl/MonitorControl#900] [Q&A]
Hi all,

After delving into DDC in order to control my monitor's audio output with my shiny new MacBook Air M1, I learned that some USB-C hubs with HDMI input support DDC, and others don't.

Unfortunately the one I bought seems NOT to support passing DDC from the monitor HDMI to the MacBook.

Does anyone know which USB-C hubs or HDMI adaptors support passing through DDC, or if there's a way to know in advance? E.g. if the hub supports HDMI 2.0, will that guarantee that DDC is also supported? It's such an obscure feature that it doesn't ever seem to be specified in product specs...

**Comment by waydabber** [comment]
@manuelkuhs - good luck with this. :) It is indeed an obscure feature. DisplayPort and HDMI relays i2c commands in a different manner so the chip in the dock must be actively support translating this. Actually the built-in DP->HDMI chip found in some Macs since 2018 does not support DDC data passthru (that's why HDMI out does not work on the Minis or the new MBP...). It would indeed be great to have a list of compatible hardware.

**Reply by ssbarnea** [reply]
If someone has success with particular cable, please post link here, so we can order one.

**Comment by OrionTheDev** [comment]
Hello, I share some information that may help you.
I use a Dell DA300 with my Macbook Air M1. I can control volume and brightness from my keyboard.

Unfortunately, PD charging is not included in the DA300 adapter.
I hope to find another model including it.

**Comment by ElhemEnohpi** [comment]
I can tell you another one that *doesn't* work: i-tec "USB-C Metal Nano dock 4k HDMI with LAN", part number C31NANODOCKLANPD - returning it to the shop now.

There are reports in [Lunar - Frequently Asked Questions](https://lunar.fyi/faq#brightness-not-changing) of a couple of docks that are supposed to work. There's a [report on Reddit](https://www.reddit.com/r/UsbCHardware/comments/i8136t/is_there_any_way_to_tell_if_a_usb_c_dock_supports/) of one that's also supposed to work, though it's not really clear which model.

Otherwise, I can't find any list of docks/hubs or HDMI adapters that are known to work, or not, with DDC and MonitorControl.
